apps/mark-scan/frontend/src/api.ts

```diff
@@ -167,14 +167,33 @@ export const getApplicationDiskSpaceSummary = {
   },
 } as const;
 
+export const getMostRecentPaperHandlerDiagnostic = {
+  queryKey(): QueryKey {
+    return ['getMostRecentPaperHandlerDiagnostic'];
+  },
+  useQuery() {
+    const apiClient = useApiClient();
+    return useQuery(
+      this.queryKey(),
+      () =>
+        apiClient.getMostRecentDiagnostic({
+          diagnosticType: 'mark-scan-paper-handler',
+        }),
+      { cacheTime: 0 }
```

Comment:
Why `cacheTime: 0`? And why only for paper handler diagnostic, not accessible controller diagnostic?

Reply:
I'm not sure the cause but this param might be redundant with the hard refetch discussed here: https://github.com/votingworks/vxsuite/pull/4752#discussion_r1576978686

I can remove this and verify no stale data is rendered.

Reply:
Ok, alternatively you could use this instead of the hard refetch. Not sure if it matters one way or the other

Reply:
[bc6e914](https://github.com/votingworks/vxsuite/pull/4891/commits/bc6e9149a1f1baa5b7e5a4efe669bc3bf495bae5)